components/Welcome/index.js

```diff
@@ -22,28 +24,103 @@ const StyledPararaph = styled.p`
   text-align: center;
   opacity: 0.8;
 `;
+const StyledLink = styled(Link)`
+  color: white;
+`;
+const StyledLoginText = styled.button`
+  background: none;
+  border: none;
+  padding: 0;
+  font-size: inherit;
+  cursor: pointer;
+  color: white;
+  text-decoration: underline;
+`;
 
 export function Welcome() {
-  return (
-    <>
-      <WelcomeWrapper>
-        <StyledHeading>Herzlich Willkommen bei Eatable</StyledHeading>
-        <p>
-          Diese App dient dazu Nahrungsmittel mit einem simplen Barcodescan
-          <br />
-          nach unerwünschten Zusatzstoffen/
-          <br />
-          Additiven oder bestimmten Allergenen zu durchsuchen.
-          <br />
-          wählen Sie in den Einstellungen Ihr &quot;Kryptonit&quot; .
-        </p>
-      </WelcomeWrapper>
-      <StyledPararaph>
-        Es besteht immer die Möglichkeit, dass Daten über Allergene fehlen,
-        unvollständig oder falsch sind oder dass sich die Zusammensetzung des
-        Produkts geändert hat. Wenn Sie allergisch sind, überprüfen Sie immer
-        die Angaben auf der tatsächlichen Produktverpackung.
-      </StyledPararaph>
-    </>
-  );
+  const { data: session } = useSession();
+
+  const convertToProperName = (name) => {
+    if (!name) return "";
+    const splitName = name.split(" ");
+    let properName = "";
+    splitName.forEach((word) => {
+      properName += word[0].toUpperCase() + word.slice(1).toLowerCase() + " ";
+    });
+    return properName.trim();
+  };
+
+  function Disclaimer() {
+    return (
+      <>
+        <StyledPararaph>
+          Es besteht immer die Möglichkeit, dass Daten über Allergene fehlen,
+          unvollständig oder falsch sind oder dass sich die Zusammensetzung des
+          Produkts geändert hat. Wenn Sie allergisch sind, überprüfen Sie immer
+          die Angaben auf der tatsächlichen Produktverpackung.
+        </StyledPararaph>
+      </>
+    );
+  }
+
+  session ? session.user.name : "";
+
+  if (session) {
+    return (
+      <>
+        <WelcomeWrapper>
+          <StyledHeading>
+            Herzlich Willkommen, {convertToProperName(session.user.name)} bei
+            Eatable
+          </StyledHeading>
+
+          <p>
+            Diese App dient dazu Nahrungsmittel mit einem simplen Barcodescan
+            <br />
```

Comment:
try to avoid the excessive use of linebreaks. Maybe you can give your paragraph a styling that makes the text break in a nice way?

Reply:
ok i will try to do so :)